molgenis-core-ui/src/main/java/org/molgenis/core/ui/jobs/JobsController.java

```diff
@@ -102,7 +101,7 @@ public String viewJob(
             e -> {
               Query<Entity> q =
                   dataService.query(e.getId()).ge(JobExecutionMetaData.SUBMISSION_DATE, weekAgo);
-              if (!currentUser.isSuperuser()) {
+              if (!Boolean.TRUE.equals(currentUser.isSuperuser())) {
```

Comment:
```User.isSuperuser()``` should return a boolean instead of Boolean since UserMetadata states that superuser is required.